lib/aca_entities/medicaid/ios/contracts/ssp_member__c_contract.rb

```diff
@@ -0,0 +1,700 @@
+            optional(:IsMigrantOrSeasonalFarmWorkerToggle__c).maybe(:string)
+            optional(:IsOnSickLeaveToggle__c).maybe(:string)
+            optional(:IsPregnantInLastThreeMonthsToggle__c).maybe(:string)
+            optional(:IsPrimaryTextPreferred__c).maybe(:string)
+            optional(:IsQualifiedNonCitizen__c).maybe(:string)
+            optional(:IsReceivingMedicareBenefitToggle__c).maybe(:string)
+            optional(:IsStateMedicaidprogramCode__c).maybe(:string)
+            optional(:IsTMember__c).maybe(:string)
+            optional(:IsToBeReferredToWicToggle__c).maybe(:string)
+            optional(:IsUSCitizenToggle__c).maybe(:string)
+            optional(:IsUSCitizenValidated__c).maybe(:string)
+            optional(:IsVerifiedLawfulPresence__c).maybe(:string)
+            optional(:IsWaitingForDecisionToggle__c).maybe(:string)
+            optional(:LifeTimeQuarterQuantity__c).maybe(:integer)
+            optional(:MaidCardNumber__c).maybe(:string)
+            optional(:MailingCertifiedNoStatIndicator__c).maybe(:string)
+            optional(:PhysicalCertifiedNoStatIndicator__c).maybe(:string)
+            optional(:MailingCotVacancyIndicator__c).maybe(:string)
+            optional(:PhysicalCotVacancyIndicator__c).maybe(:string)
+            optional(:MailingMailAddrDeliveryCode__c).maybe(:string)
+            optional(:PhysicalMailAddrDeliveryCode__c).maybe(:string)
+            optional(:MailingResidentialDeliveryIndicator__c).maybe(:string)
+            optional(:PhysicalResidentialDeliveryIndicator__c).maybe(:string)
+            optional(:MCIId__c).maybe(:string)
+            optional(:MiddleInitial__c).maybe(:string)
+            optional(:NationalityCode__c).maybe(:string)
+            optional(:NaturalizationNumber__c).maybe(:string)
+            optional(:NonCitizenClassOfAdmissionCode__c).maybe(:string)
+            optional(:NonCitizenEntryDate__c).maybe(:date)
+            optional(:NoReasonSSNCode__c).maybe(:string)
+            optional(:NumberOfBirthsExpected__c).maybe(:integer)
+            optional(:OrganizationName__c).maybe(:string)
+            optional(:ParentUnableToCareForChildToggle__c).maybe(:string)
+            optional(:PassportNumber__c).maybe(:string)
+            optional(:PhysicalAddressLine2__c).maybe(:string)
+            optional(:PhysicalAddrValidated__c).maybe(:string)
+            optional(:MailingAddrValidated__c).maybe(:string)
+            optional(:PhysicalCountyCode__c).maybe(:string)
+            optional(:PhysicalZipCode4__c).maybe(:string)
+            optional(:MailingAddressLine1__c).maybe(:string)
+            optional(:MailingAddressLine2__c).maybe(:string)
+            optional(:MailingCity__c).maybe(:string)
+            optional(:MailingCountyCode__c).maybe(:string)
+            optional(:MailingStateCode__c).maybe(:string)
+            optional(:MailingZipCode4__c).maybe(:string)
+            optional(:MailingZipCode5__c).maybe(:string)
```

Comment:
this is marked as required on the database, shall we set as required here ?

Reply:
These are required if you're building a full address, we wouldn't be doing so if they had the same mailing and physical address. I'll add conditional validations to that effect

Reply:
Actually, I already wrote those validations, they're [already in the code base](lib/aca_entities/medicaid/ios/contracts/ssp_member__c_contract.rb) lines 660-693.